shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -2,18 +2,27 @@
 // Licensed under the MIT License.
 "use strict";
```

Comment:
Just curious, for all of the helper functions in here, do you think there would be any benefit to adding a compat check so that it calls the native version if it is available? Would this provide any perf benefit?

Reply:
I did consider this and decided against it for 2 reasons
1) It adds extra code to the overall package
2) See the Note on the polyfils: Note: For consistency this will not use the Array.prototype.xxxx implementation if it exists as this would
     * cause a testing requirement to test with and without the implementations